### DISCUSSION TITLE
Unsplit terminal

### DISCUSSION THREAD

**Discussion by Serene7n** [Eugeny/tabby#6927] [Q&A]
How do you unsplit a terminal in tabby after using Ctel+Shift+d?

**Comment by Eugeny** [comment]
a) close one of the panes (Ctrl-D or "exit")
b) hold Ctrl/Cmd-Shift (the "rearrange panes" shortcut) and drag a pane into a separate tab

**Comment by Serene7n** [comment]
thanks

**Comment by sIRwa2** [comment]
but what if you dont want to close , just have it move back to beeing a tab? above options dont seem to do that.

Aaaa, its the very small "handle" at the far left of the connection title that you need to grab.. lessons learned
